crates/control/src/behavior/initial.rs

```diff
@@ -47,6 +50,19 @@ fn look_at_referee(
         return None;
     };
 
+    let own_team_is_home_after_coin_toss = world_state
+        .filtered_game_controller_state?
+        .own_team_is_home_after_coin_toss;
+
+    match (
+        world_state.robot.player_number,
```

Comment:
The idea (which was actually not implemented in this PR) was that the positions on which sides the robots start in `Initial` are given by the player number and not the role, if I'm not mistaken. Because we want to see the referee on the other side of the field and not next to us, and also only the inner most two robots should look there.

Reply:
Okay, then let's leave it like this